operators/pkg/instance-controller/container_logic.go

```diff
@@ -0,0 +1,175 @@
+package instance_controller
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+	crownlabsv1alpha2 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha2"
+	instance_creation "github.com/netgroup-polito/CrownLabs/operators/pkg/instance-creation"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func CreateGUIContainerPod(name string, namespace string, template *crownlabsv1alpha2.Environment, instanceName string, references []metav1.OwnerReference, examMode bool) (appsv1.Deployment, error) {
+
+	var examModeStr string
+	if examMode {
+		examModeStr = "true"
+	} else {
+		examModeStr = "false"
+	}
```

Comment:
Should be replaced with [FormatBool](https://golang.org/pkg/strconv/#FormatBool)